--- tolua++-1.0.93/config_linux.py~	2005-11-30 23:31:27.000000000 +0100
+++ tolua++-1.0.93/config_linux.py	2010-11-08 14:34:33.795188003 +0100
@@ -4,7 +4,7 @@
 
 # Compiler flags (based on Debian's installation of lua)
 #LINKFLAGS = ['-g']
-CCFLAGS = ['-I/usr/include/lua50', '-O2', '-ansi', '-Wall']
+CCFLAGS = ['-I/usr/include/lua51', '-O2', '-ansi', '-Wall']
 #CCFLAGS = ['-I/usr/include/lua50', '-g']
 
 # this is the default directory for installation. Files will be installed on
@@ -15,8 +15,8 @@
 #
 # You can see more 'generic' options for POSIX systems on config_posix.py
 
-prefix = '/usr/local'
+prefix = '/usr'
 
 # libraries (based on Debian's installation of lua)
-LIBS = ['lua50', 'lualib50', 'dl', 'm']
+LIBS = ['lua51', 'dl', 'm']
 
